src/System.IO.Packaging/src/System/IO/Packaging/Package.cs

```diff
@@ -837,9 +837,9 @@ internal static void ThrowIfCompressionOptionInvalid(CompressionOption compressi
 
             if (packageMode == FileMode.OpenOrCreate && packageAccess != FileAccess.ReadWrite)
                 throw new ArgumentException(SR.UnsupportedCombinationOfModeAccess);
-            if (packageMode == FileMode.Create && packageAccess != FileAccess.ReadWrite)
+            if (packageMode == FileMode.Create && ((packageAccess & FileAccess.Write) == 0))
```

Comment:
this is ok, or you could delete the check as FileStream will still throw an ArugmentException.

Reply:
yeah i'll delete the check. i thought the test was no longer reporting an `ArgumentException` but then i realized i was wrong